modin/core/dataframe/algebra/binary.py

```diff
@@ -36,6 +123,14 @@ def register(cls, func, join_type="outer", labels="replace"):
         labels : {"keep", "replace", "drop"}, default: "replace"
             Whether keep labels from left Modin DataFrame, replace them with labels
             from joined DataFrame or drop altogether to make them be computed lazily later.
+        how_compute_dtypes : {"common_cast", "float","bool", None}, default: None
+            How dtypes should be inferred.
+                * If "common_cast", casts to common dtype of operand columns.
+                * If "float", performs type casting by finding common dtype.
+                   If the common dtype is any of the integer types, perform type casting to float.
+                   Used in case of truediv.
```

Comment:
```suggestion
                * If "float", performs type casting by finding common dtype.
                  If the common dtype is any of the integer types, perform type casting to float.
                  Used in case of truediv.
```

Reply:
done.